pkg/tests/ossm/prometheus_scoped_scraping_test.go

```diff
@@ -0,0 +1,134 @@
+package ossm
+
+import (
+	"fmt"
+	"math/rand"
+	"testing"
+
+	"github.com/maistra/maistra-test-tool/pkg/util/check/common"
+	"github.com/maistra/maistra-test-tool/pkg/util/check/require"
+	"github.com/maistra/maistra-test-tool/pkg/util/env"
+	"github.com/maistra/maistra-test-tool/pkg/util/hack"
+	"github.com/maistra/maistra-test-tool/pkg/util/oc"
+	"github.com/maistra/maistra-test-tool/pkg/util/retry"
+	"github.com/maistra/maistra-test-tool/pkg/util/shell"
+	"github.com/maistra/maistra-test-tool/pkg/util/test"
+)
+
+func TestOperatorCanUpdatePrometheusConfigMap(t *testing.T) {
+	test.NewTest(t).Groups(test.Full).Run(func(t test.TestHelper) {
+		hack.DisableLogrusForThisTest(t)
+
+		t.Log("This test checks if the operator can update Prometheus ConfigMap when the SMMR is updated")
+
+		meshNamespace := env.GetDefaultMeshNamespace()
+		defaultSMMR := shell.Executef(t, "oc -n %s get smmr default -o jsonpath='{.metadata.annotations.kubectl\\.kubernetes\\.io/last-applied-configuration}'", meshNamespace)
+
+		t.Cleanup(func() {
+			oc.ApplyString(t, meshNamespace, defaultSMMR)
```

Comment:
smmr default template can be get from yaml_config.go in ossm package and be reinstalled like this:
`oc.ApplyString(t, meshNamespace, smmr)`